packages/cloud-cognitive/src/components/PageHeader/PageHeader.js

```diff
@@ -742,6 +744,12 @@ PageHeader.propTypes = {
    * a breadcrumb. Optional.
    */
   breadcrumbItems: PropTypes.element, // expects BreadcrumbItems,
+  /**
+   * If the user supplies breadcrumbs then they must supply an aria label for the overflow
+   */
+  breadcrumbOverflowLabel: PropTypes.string.isRequired.if(
+    ({ breadcrumbItems }) => breadcrumbItems && breadcrumbItems.length > 0
```

Comment:
Ooo I didn't know about this type of validation, we could clean up some other components with this approach